src/IO/InterfileHeader.cxx

```diff
@@ -1032,7 +1032,7 @@ InterfilePDFSHeader::post_processing()
         {
           warning("Inconsistent number of TOF bins (" + std::to_string(this->num_timing_poss)
                   + ") and size of the 'TOF bin order' list (" + std::to_string(this->timing_poss_sequence.size()) + ").");
-          // return true;
+          return true;
```

Comment:
@NicoleJurjew I've made this into a failure. (Seems weird to try and go ahead of the sizes are incorrect.). Let me know ASAP if this is not ok.

Reply:
Hey Kris, I agree, that wouldn't make sense. With the new script we're not manually editing the TOF bin order anymore, so this won't trip us up for Vision files.